static/app/views/organizationGroupDetails/actions/index.tsx

```diff
@@ -482,13 +482,18 @@ class Actions extends Component<Props, State> {
 }
 
 const Wrapper = styled('div')`
-  display: grid;
+  display: flex;
+  flex-wrap: wrap;
   justify-content: flex-start;
   align-items: center;
   grid-auto-flow: column;
-  gap: ${space(0.5)};
   margin-top: ${space(2)};
+  margin-bottom: -${space(0.5)};
   white-space: nowrap;
+
+  > * {
+    margin: 0 ${space(0.5)} ${space(0.5)} 0;
```

Comment:
Let’s just use gap :shrug:

Reply:
Thanks, forget that I can use that now 😄